lib/helpers/verify-component-specification-version.js

```diff
@@ -8,7 +8,7 @@ const { getOrigamiJson } = require('./files');
  */
 module.exports = async () => {
 	const origamiJson = await getOrigamiJson();
-	if (!origamiJson || typeof origamiJson.origamiVersion !== 'number') {
+	if (!origamiJson || typeof origamiJson.origamiVersion !== 'string') {
```

Comment:
There is a condition after this which throws a more helpful error message when run obt is run on a v1 component, which won't be hit anymore

Reply:
thanks, i transposed them